CA-31553: Mark vswitch logs as "missingok"

These logs are not present if vswitch is not enabled.
diff -r 4526b11d3842 xenserver/etc_logrotate.d_vswitch
--- a/xenserver/etc_logrotate.d_vswitch	Fri Oct 02 15:21:00 2009 +0100
+++ b/xenserver/etc_logrotate.d_vswitch	Mon Oct 05 09:39:00 2009 +0100
@@ -7,6 +7,7 @@
 
 /var/log/ovs-vswitchd.log /var/log/ovs-brcompatd.log {
 	sharedscripts
+	missingok
 	postrotate
 	# Tell ovs-vswitchd and ovs-brcompatd to reopen their log files
 	/sbin/service vswitch reload
